lib/src/settings.rs

```diff
@@ -66,6 +66,7 @@ pub struct GitSettings {
     #[cfg(feature = "git2")]
     pub subprocess: bool,
     pub executable_path: PathBuf,
+    pub change_id: bool,
```

Comment:
Maybe this should be called `git.write-change-id-header` or something?

Reply:
I change it to `change-id-header` to make it more clear. The `write-` prefix seems a bit misleading to me since it's also about loading the change-id from the commit object.

Reply:
Don't we load the `change-id` header no matter if the write path is enabled or not? (That's why I suggested `write-` instead of `use-`.)